src/Facemesh/index_test.js

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2020 ml5
+//
+// This software is released under the MIT License.
+// https://opensource.org/licenses/MIT
+
+const { facemesh } = ml5;
+
+const FACEMESH_IMG = 'https://cdn.jsdelivr.net/gh/ml5js/ml5-data-and-models@master/tests/images/harriet_128x128.jpg';
+
+describe('Facemesh', () => {
+  let facemeshInstance;
+  let testImage;
+
+  async function loadHTMLImageElement(imageSrc) {
```

Comment:
as side note: we should extract out these loadImage functions in to some kind of testing utils module :)

Reply:
for another PR though. Let's keep this for now.